renovate.json

```diff
@@ -0,0 +1,40 @@
+{
+    "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+    "extends": [
+      "config:recommended"
+    ],
+    "ignorePaths": [],
+    "labels": ["Skip Changelog", "dependencies"],
+    "postUpdateOptions" : [
+      "gomodTidy"
+    ],
+    "postUpdateTask" : {
+        "commands": ["make", "fixtures"]
+    },
```

Comment:
FYI this won't work unless we use self-hosted renovate https://docs.renovatebot.com/configuration-options/#postupgradetasks:~:text=Post%2Dupgrade%20tasks%20can%20only%20be%20used%20on%20self%2Dhosted%20Renovate%20instances.

Reply:
Got it.  Lets skip for now.

Reply:
removed